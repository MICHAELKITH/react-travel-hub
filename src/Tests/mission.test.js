import { Provider } from 'react-redux';
import TestRenderer from 'react-test-renderer';
import Missions from '../components/mission';
import Store from '../slices/store';

describe('Missions', () => {
  test('Render Missions Component', () => {
    const tree = TestRenderer.create(
      <Provider store={Store}>
        <Missions />
      </Provider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
