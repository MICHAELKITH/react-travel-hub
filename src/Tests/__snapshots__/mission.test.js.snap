// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Missions Render Missions Component 1`] = `
<div>
  <table
    border={1}
    className="header_table"
  >
    <tr>
      <th
        className="header_mission"
      >
        Mission
      </th>
      <th
        className="header_description"
      >
        Description
      </th>
      <th
        className="header_status"
      >
        status
      </th>
      <th
        aria-label="empty"
        className="header_action"
      />
    </tr>
  </table>
</div>
`;
